app/styles/_mobile-clients.less

```diff
@@ -0,0 +1,41 @@
+.mobile-clients {
```

Comment:
Please keep the declarations sorted alphabetically in this file. We use this style guide:

http://codeguide.patternfly.org/#css-declaration-order